src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationService.java

```diff
@@ -48,17 +48,6 @@ public void notifyClaimant(final Claim claim) {
         );
     }
 
-    public void notifyDefendant(final Claim claim) {
-        final Map<String, String> parameters = aggregateParams(claim);
-        sendNotificationEmail(
-            claim.getClaimData().getDefendant().getEmail()
-                .orElseThrow(() -> new NotificationException(("Unknown defendant email"))),
-            notificationsProperties.getTemplates().getEmail().getDefendantCCJRequested(),
-            parameters,
-            NotificationReferenceBuilder.CCJRequested.referenceForDefendant(claim.getReferenceNumber())
```

Comment:
Don't we have to remove `NotificationReferenceBuilder.CCJRequested.referenceForDefendant` method as well?